public/approval/v0.0.1/openapi.json

```diff
@@ -1817,13 +1821,17 @@
                 "allOf": [
                     {
                         "$ref": "#/components/schemas/UserIn"
+                    },
+                    {
+                        "type": "object",
+                        "description": "Onput parameters for User object",
```

Comment:
In https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.0.md I saw `"description"` is at top level parallel with `"offOf"`.

Reply:
Yes, the `description` will be automated skipped when creating client side docs.

Reply:
let's move to top for better readability.

Reply:
Changed.